### DISCUSSION TITLE
Arrows Design

### DISCUSSION THREAD

**Discussion by francoRemitee** [FlowmapBlue/FlowmapBlue#59] [Q&A]
Hi, would like to know if there is any way to customize the arrows design.
Thank You

**Comment by ilyabo** [comment]
Hi Franco,
Could you elaborate on what kind of design you are looking for and for what purpose?

**Comment by francoRemitee** [comment]
Hi ilyabo, yes. 
First of all, congratulations. This is an amazing tool. Beautiful visualizations.
The design for the arrows i was thinking about is something like this https://w7.pngwing.com/pngs/95/514/png-transparent-red-turning-arrow-gules-arrow-design.png
Maybe not curved if its a problem. This is for esthetics.
In terms of functionallity: It's possible to set a maximum size for the arrows? the size of the arrows scales with the represented values, maybe there is a way to shrink the range of sizes for the arrows?

**Reply by francoRemitee** [reply]
For the second question: Maybe i could use logarithmic scale for the values, but it would showcase this values when panning over the arrow.

**Reply by ilyabo** [reply]
Yes, I am hoping to add curved arrows some time soon.

> the size of the arrows scales with the represented values

If you want to disable the auto-scaling there's the switch "Dynamic range adjustment" for that

**Reply by francoRemitee** [reply]
What's the "Dynamic range adjustment" switch supposed to change? i'm not noticing any diference.
![image](https://user-images.githubusercontent.com/100226081/155365642-8a56dcac-fa24-4128-a498-a06ccad3af7d.png)
![image](https://user-images.githubusercontent.com/100226081/155365706-c1dcd9d7-5657-4bc7-ba36-645a88f8e9cc.png)

**Comment by ilyabo** [comment]
Try zooming in to an area with smaller flows. If you disable it, the scales will not be adjusted to make smaller flows' look big when you zoom in.

**Reply by francoRemitee** [reply]
ooh, ok. I see it now. What i meant was that if you filter to get only one flow it always has the same size (as it is always the maximum), but it does not reflect its progress in time. I know it's tricky to decide wich value should the maximum be, maybe it could show the arrow's size as if the other flows where present.

**Reply by ilyabo** [reply]
Ah, got it now. You meant the scales over time. Yes, I would like to fix this. It's tricky, but I have some ideas how I could do it. Hope to do it some time soon.
